site/zenodo_rdm/stats/config.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Configuration for ZenodoRDM stats."""
+
+STATS_PIWIK_EXPORTER = {
```

Comment:
Question: @slint Should this be in `invenio.cfg`? I don't think it matters since env vars will take precedence.

Reply:
It's ok for it to stay here, also to keep the module/logic somewhat isolated. Indeed we'll override it eventually via environment on OpenShift.